ember-cli-build.js

```diff
@@ -68,6 +68,14 @@ module.exports = function (defaults) {
             optional: ['es6.spec.symbols'],
             includePolyfill: true
         },
+        'ember-service-worker': {
+            rootUrl: '/ghost/'
```

Comment:
Offline wouldn't work, but it wouldn't break things. We (for now) need to know ahead of compile-time where we're running.